<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppPresent</class>
 <widget class="QMainWindow" name="AppPresent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>625</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Song Presentation</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../icons/icons.qrc">
    <normaloff>:/icons/app_icon</normaloff>:/icons/app_icon</iconset>
  </property>
  <widget class="QWidget" name="WidgetCentral">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>0</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="WrapperLayout">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="MainLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="HeadLayout">
          <item>
           <widget class="QLabel" name="LblApp">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Trebuchet MS</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string> mBible version2.5.5</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="BtnClose">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Trebuchet MS</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>X</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="TopLayout">
          <item>
           <widget class="QLabel" name="TxtReading">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Trebuchet MS</family>
              <pointsize>18</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="LblChapterNo">
            <property name="text">
             <string>CHAPTER:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="CmbChapterNo">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Trebuchet MS</family>
              <pointsize>15</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="editable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="LblVerseNo">
            <property name="text">
             <string>VERSE:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="CmbVerseNo">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Trebuchet MS</family>
              <pointsize>15</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="currentText">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolBar" name="ToolbarMain">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="windowTitle">
             <string notr="true">toolBar1</string>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
            <addaction name="actionPrevious"/>
            <addaction name="actionNext"/>
            <addaction name="actionBigger"/>
            <addaction name="actionSmaller"/>
            <addaction name="actionBold"/>
            <addaction name="actionFont"/>
            <addaction name="actionTheme"/>
            <addaction name="actionClose"/>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="MidLayout">
          <item>
           <widget class="QPlainTextEdit" name="TxtScripture">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="verticalScrollBarPolicy">
             <enum>Qt::ScrollBarAsNeeded</enum>
            </property>
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionPrevious">
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/icons/last.png</normaloff>:/icons/last.png</iconset>
   </property>
   <property name="text">
    <string> </string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/icons/next.png</normaloff>:/icons/next.png</iconset>
   </property>
   <property name="text">
    <string> </string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string/>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionBigger">
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/icons/bigger.png</normaloff>:/icons/bigger.png</iconset>
   </property>
   <property name="text">
    <string> </string>
   </property>
   <property name="shortcut">
    <string>=</string>
   </property>
  </action>
  <action name="actionSmaller">
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/icons/smaller.png</normaloff>:/icons/smaller.png</iconset>
   </property>
   <property name="text">
    <string> </string>
   </property>
   <property name="shortcut">
    <string>-</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/icons/font.png</normaloff>:/icons/font.png</iconset>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="shortcut">
    <string>F</string>
   </property>
  </action>
  <action name="actionBold">
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/icons/bold.png</normaloff>:/icons/bold.png</iconset>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="shortcut">
    <string>B</string>
   </property>
  </action>
  <action name="actionTheme">
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/icons/view</normaloff>:/icons/view</iconset>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="shortcut">
    <string>T</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
